src/main/java/Bing.java

```diff
@@ -0,0 +1,144 @@
+import java.util.Scanner;
+import java.util.ArrayList;
+public class Bing {
+    public static void main(String[] args) {
```

Comment:
Consider abstracting away portions of your code to helper methods and then refactoring your code to match the OOP style of coding. Kudos on the extensive error handling nonetheless!